lib/Rex/CLI.pm

```diff
@@ -761,13 +758,16 @@ sub load_rexfile {
     # update %INC so that we can later use it to find the rexfile
     $INC{"__Rexfile__.pm"} = $rexfile;
 
-    # reopen STDERR
-    open STDERR, ">&", $default_stderr;
-
-    if ($stderr) {
-      my @lines = split( $/, $stderr );
+    if (@warnings) {
       Rex::Logger::info( "You have some code warnings:", 'warn' );
-      Rex::Logger::info( "\t$_",                         'warn' ) for @lines;
+      for (@warnings) {
+
+        # remove /loader/.../ prefix before filename
+        ## no critic (RegularExpressions);
+        s|/loader/[^/]+/||;
```

Comment:
I believe strictly speaking this line is an independent change in behavior ("don't hijack STDERR while loading Rexfiles" vs "make the output more human-friendly").

I like the cleanup, especially since it also being tested, but it might work better to split this change out into a separate commit.